openquake/hazardlib/tests/geo/surface/kite_fault_test.py

```diff
@@ -0,0 +1,409 @@
+# The Hazard Library
+# Copyright (C) 2020 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import glob
+import unittest
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D  # This is needed
+from openquake.hazardlib.geo import Point, Line
+from openquake.hazardlib.geo.geodetic import distance
+from openquake.hazardlib.geo.surface import KiteSurface
+
+from typing import Optional
+
+BASE_DATA_PATH = os.path.join(os.path.dirname(__file__), 'data')
+PLOTTING = False
+
+
+def ppp(profiles: list, smsh: KiteSurface, title: Optional[str] = ''):
+
```

Comment:
`Optional` mean that title can be None. Is that what you mean? Because it looks to me that it can be the empty string but not None.

Reply:
You're right Michele I fixed it